test/testdrive/storage-clusters.td

```diff
@@ -22,7 +22,7 @@ ALTER SYSTEM SET enable_unmanaged_cluster_replicas = true
 
 # Querying a cluster with no replicas does not succeed.
 > SET cluster = storage
-! SELECT 1
+! SELECT generate_series(1, 1)
```

Comment:
@andrewrodriguez-m I don't understand why this change is required. Both the new and the previous are optimized to the constant singleton collection `{{ 1 }}`. Consequently, I would expect that both variants should reduce to a `FastPathPlan::Constant` which does not need a cluster to execute something before it returns.

```
mz> EXPLAIN SELECT generate_series(1, 1)
+------------------------------+
| Optimized Plan               |
|------------------------------|
| Explained Query (fast path): |
|   Constant                   |
|     - (1)                    |
+------------------------------+
SELECT 1
Time: 0.038s
mz> EXPLAIN SELECT 1
+------------------------------+
| Optimized Plan               |
|------------------------------|
| Explained Query (fast path): |
|   Constant                   |
|     - (1)                    |
+------------------------------+
SELECT 1
Time: 0.032s
```